otj-metrics-core/src/main/java/com/opentable/metrics/graphite/OtGraphiteReporter.java

```diff
@@ -365,8 +367,11 @@ private void reportTimer(String name, Timer timer, long timestamp) throws IOExce
         reportMetered(name, timer, timestamp);
     }
 
+    /**
+     * CHANGE: call {@link OtGraphiteReporter#reportCounter(String, Counter, long)}
```

Comment:
What is this JavaDoc trying to say?

Reply:
That we inserted call to the reportCounter() ;)

Reply:
It is attempt to document changes to the original class

Reply:
It sounds to me like a TODO, i.e. this should be changed. Can you rephrase it?

Reply:
Done